.github/pull_request_template.md

```diff
@@ -1,6 +1,8 @@
 #### Description of changes
 
-(give an overview. Add technical description describing your changes)
+<!-- give an overview. Add technical description describing your changes -->
+
+<!-- Please use a suitable tag (feat, fix, chore, refactor) on the PR message or PR title to help in release validation and release notes -->
```

Comment:
Will external contributors know the differences? It might be worth having these called out in a markdown file, then refer to the file from within the prompt

Reply:
Updated the checklist item to make it more clear. I feel like creating a markdown to explain this might be an overkill, but thats just my opinion.

Reply:
I don't think documenting it would be overkill; axe-core, for example, has a [code-submission-guidelines.md](https://github.com/dequelabs/axe-core/blob/develop/doc/code-submission-guidelines.md#how-to-structure-your-commits) document that does this for their repo.

In axe-sarif-converter, we have the template refer to the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) site for more information, but if we wanted to suggest a specific `flagged:` prefix, we'd probably want our own summary. A new section in `/docs/workflow.md` would be reasonable.

Reply:
Updated `workflow.md`. Please check if thats what you had in mind.